### DISCUSSION TITLE
build-logic vs buildSrc after Gradle 8

### DISCUSSION THREAD

**Discussion by ersen-lw** [android/nowinandroid#962] [Q&A]
Hello,

I'm in the process of migrating to Kotlin Gradle DSL. Currently, I have my custom plugins in `buildSrc`. Working fine so far but I ran into one of the pitfalls where a single change in `buildSrc` causes a full build. I discovered this project and `build-logic` which looks like a better alternative. 

I saw this PR https://github.com/gradle/gradle/pull/22540 and they make `buildSrc` behave like a included-build which looks to be a solution to the issue I mentioned above. 

Using Gradle 8.1 , I still encounter that issue where a single change in `buildSrc` (I just update one dependency constant) invalidates the entire build so I'm not sure what that PR changes. 

Anyway, my question is would you still recommend using `build-logic` over `buildSrc` if you were to create a new project today ?

**Comment by Keith-Albright-Bose** [comment]
There's a description of the two build setups here: 
https://docs.gradle.org/current/userguide/intro_multi_project_builds.html#sec:project_standard
